raiden/smart_contracts/NettingChannelContract.sol

```diff
@@ -48,10 +51,15 @@ contract NettingChannelContract {
         return success;
     }
 
+    /// @notice Get the address of the channel partner.
+    /// @param one_address The address of partner 1.
```

Comment:
It's not your fault but I think we should change the parameter name here. Can you do it?

I would name it `address participant_address`.
So the function parameter is the participant's address and the `@return` should return the address of the partner of that participant. 

Can you edit the docstring and the function to reflect that?